src/main/java/com/dataloom/datastore/permissions/controllers/PermissionsController.java

```diff
@@ -44,7 +44,7 @@
     @RequestMapping(
```

Comment:
Please ensure your mappings are consistent with what is in other API and controllers.